internal/cephfs/volume.go

```diff
@@ -119,9 +119,17 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 	return &subvol, nil
 }
 
+type resizeState int64
+
+const (
+	resizeUnknown resizeState = iota
+	resizeSupported
+	resizeUnsupported
+)
+
```

Comment:
I would like to change this to operationState and below const to represent an operation which might be helpful when we need the same for other operations

```go
type operationState int64

const (
	unknown operationState = iota
	supported
	unsupported
)
```